docs/vgclog.md

```diff
@@ -0,0 +1,269 @@
+
+Splitting the global cycle operations into these increments reduces pause times by running most of the GC operations concurrently with application threads. The `gencon` global cycle's concurrent increment is paused when a `gencon` partial GC cycle is triggered and resumes when the partial cycle(s) completes. In this way,a global cycle can progress while other types of cycle are run by pausing and resuming the concurrent work. In some policies, concurrent operations are split further into *multiple* concurrent increments for better control of progress of the concurrent operation.
+
+You can see this interleaving of the increments in the verbose GC log. The following table illustrates how the different increments of an example log interleave for the `gencon` policy (for clarity, not all GC activities are listed):
+
+<table style="width:100%">
+
+  <tr>
+    <th>Line Number</th>
+    <th colspan="2">Logging</th>
+  </tr>
+  <tr>
+    <th>-</th>
+    <th>`gencon` global GC cycle status</th>
+    <th>`gencon` partial GC cycle status</th>
+  </tr>
+  <tr>
+    <td>1-87</td>
+    <td colspan="2">initialization</th>
+  </tr>
+  <tr>
+    <td>87-51676</td>
+    <td> - </td>
+    <td> series of `gencon` partial cycles start and finish </td>
+  </tr>
+
+  <tr>
+    <td>51677</td>
+    <td> global cycle's initial increment marked</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51680</td>
+    <td> STW pause starts</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51683</td>
+    <td> global cycle starts</td>
+    <td>-</td>
+  </tr>
+
+    <tr>
+    <td>51684</td>
+    <td> STW pause ends</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51709</td>
+    <td> concurrent increment runs (blank line)</td>
+    <td>-</td>
+  </tr>
+  <tr>
+    <td>51686</td>
+    <td> concurrent increment paused</td>
+    <td>STW pause starts</td>
+  </tr>
+    <tr>
+    <td>51690</td>
+    <td> concurrent increment paused</td>
+    <td>partial cycle starts</td>
+  </tr>
+
+  <tr>
+    <td>51708</td>
+    <td> concurrent increment paused</td>
+    <td>scavenge increment runs</td>
+  </tr>
+
+  <tr>
+    <td>51730</td>
+    <td> concurrent increment paused</td>
+    <td>partial cycle ends</td>
+  </tr>
+
+  <tr>
+    <td>51733</td>
+    <td> concurrent increment resumes</td>
+    <td>STW pause ends</td>
+  </tr>
+
+      <tr>
+    <td>51734</td>
+    <td> concurrent increment finishes</td>
+    <td> - </td>
+  </tr>
+
+  <tr>
+    <td>51735</td>
+    <td> STW pause starts</td>
+    <td></td>
+  </tr>
+
+  <tr>
+    <td>51738</td>
+    <td> final global increment runs </td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>51793</td>
+    <td> global cycle ends </td>
+    <td></td>
+  </tr>
+
+  <tr>
+    <td>51795</td>
+    <td> STW pause ends</td>
+    <td></td>
+  </tr>
+</table>
+
+**Note:** More than one partial gc cycle might run between the start and end of a `gencon` global GC cycle
```

Comment:
More than one or none partial gc...

Reply:
Zero, one, or multiple gc cycles might run between the start and end of a `gencon` global GC cycle.